tests_new/integration_tests/modules/dashboards/conftest.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from integration_tests.modules.dashboards.mutations import (
+    import_dashboard,
+    delete_dashboard,
+    request_dashboard_share,
+    reject_dashboard_share,
+)
+from integration_tests.modules.dashboards.queries import get_dashboard
+from integration_tests.core.environment.utils import set_env_params
+
+
+def create_dataall_dashboard(client, session_id, dashboard_id, env):
+    dashboard_input = {
+        'label': session_id,
+        'dashboardId': dashboard_id,
+        'environmentUri': env.environmentUri,
+        'description': 'integration test dashboard',
+        'SamlGroupName': env.SamlGroupName,
+        'tags': [session_id],
+        'terms': [],
+    }
+    ds = import_dashboard(client, dashboard_input)
+    return get_dashboard(client, ds.dashboardUri)
+
+
+@pytest.fixture(scope='session')
+def dashboard1(session_id, client1, persistent_env1, testdata):
+    set_env_params(client1, persistent_env1, dashboardsEnabled='true')
+    dashboardId = testdata.dashboards['persistent_env1'].dashboardId
```

Comment:
In the README the environment for the dashboards was the session_env1, we need to change the one here or the one there

Reply:
let's keep as `persistent_env` and I can change in README but we also do not have `persistent_env` as part of the `envs` part of json as well so I will add both

Reply:
after repsonding to comment below - may switch all to `session_env1` depending on outcome of that thread